backend/app/services/repositories/user_file_repository.py

```
@@ -0,0 +1,63 @@
+
+class UserFileRepository:
+    def __init__(self):
+        self._db_gen = get_db()
+        self.db = next(self._db_gen)
+
+    def __del__(self):
+        if hasattr(self, "_db_gen") and self._db_gen is not None:
+            try:
+                self._db_gen.close()
+            except Exception:
+                pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)